src/main/cli.js

```diff
@@ -0,0 +1,34 @@
+import { app } from 'electron'
```

Comment:
Need we add some documents in doc folder and referred in the README.md file?

Reply:
I'll will do that. See my comment above.